csharp/ql/test/library-tests/dataflow/library/FlowSummaries.expected

```diff
@@ -1,2813 +1,2925 @@
-| MS.Internal.Xml.Linq.ComponentModel.XDeferredAxis<>.GetEnumerator() | element of argument -1 -> property Current of return (normal) | true |
```

Comment:
Why do we have different number of removed and added lines? (`-1,2813 +1,2925`)

Reply:
Excellent question. The former way of printing threw away information from the summary content, which meant that some results were collapsed as they were identical.
As an example: `GetConsumingEnumerable`. In the original expected file, there was a single row in the expected file, but in the new one there are two rows. The difference between the new and old way of printing is that properties now include their qualified name, which is the new way of printing produces two strings in the resulting relation.